lib/mcp/app/resource_template.rb

```diff
@@ -53,41 +53,8 @@ def to_resource_template_hash
             mime_type: @mime_type,
             description: @description,
             handler: @handler,
-            variables: @variables
```

Comment:
The variables are easy to extract if we need them:

```ruby
Addressable::Template.new(uri_template).variables
```

However, I noticed it wasn't used at all so I removed them.